<page>
    <h1>Connect</h1>
    <config id="CONNECT_DESC"/>
    <h3>Example</h3>
    <syntax-highlight mark="8-11">
        from ntgcalls import NTgCalls
        ...

        app = NTgCalls()
        ...
        tg_params = ...

        await app.connect(
            -1001185324811,
            tg_params,
        )
    </syntax-highlight>
    <separator/>
    <h2>Details</h2>
    <category-title noref="true">
        <shi>async</shi> <ref src="method">NTgCalls.<sb>connect</sb></ref>()
    </category-title>
    <subtext>
        <config id="CONNECT_DESC2"/>
        <br/>
        <category>
            <pg-title>PARAMETERS</pg-title>
            <subtext>
                <config id="ARG_INT_ID"/>
                <category-title><ref>params</ref>: <shi>str</shi></category-title>
                <config id="TG_PARAMS_DESC"/>
            </subtext>
        </category>
        <br/>
        <category>
            <pg-title>RAISES</pg-title>
            <subtext>
                <category-title>exception <ref>ntgcalls.<sb>ConnectionError</sb></ref></category-title>
                <subtext>
                    <text>
                        This error occurs when attempting to execute an action on a non-existent Group Call.
                    </text>
                </subtext>
                <config id="TELEGRAM_EXCEPTION"/>
                <config id="RTMP_EXCEPTION"/>
                <category-title>exception <ref>ntgcalls.<sb>InvalidParams</sb></ref></category-title>
                <subtext><text>This error occurs when attempting to send invalid Transports.</text></subtext>
            </subtext>
        </category>
    </subtext>
</page>